public/scripts/toggle-util.js

```diff
@@ -160,6 +161,30 @@ var ToggleUtil = (function($, ViewUtil, StateUtil) {
     return arr;
   }
 
+    function gsReadsMapFunc(step_map) {
```

Comment:
There was a good bit of copy/pasting here and the code could definitely be generalized between HDFS and GCS, but that's for another day, or when another major file type is added.